### PR TITLE
GEN-2060: Termination survey UI

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## [APP-XXX]

- Updated logic for opening termination
- Added survey UI

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

**Review by julia-andersson — COMMENTED** [review]
https://github.com/HedvigInsurance/ugglan/assets/123949181/68186792-7ab5-4aab-ab2b-dfa6b86114c3

Maybe animations for free text field looks a bit off or what do you think?

File: Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift [hunk 1]
```diff
@@ -10,7 +10,62 @@ public class TerminateContractsOctopus: TerminateContractsService {
             input: OctopusGraphQL.FlowTerminationStartInput(contractId: contractId),
             context: nil
         )
-        return try await mutation.execute(\.flowTerminationStart.fragments.flowTerminationFragment.currentStep)
+        //        return try await mutation.execute(\.flowTerminationStart.fragments.flowTerminationFragment.currentStep)
```
Comment (julia-andersson) [inline Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift:13]: remove comment

**Review by julia-andersson — CHANGES_REQUESTED** [review]
Just that animation thing, otherwise looks good!😄

File: Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/TerminateContracts/Sources/Service/OctopusImplementation/TerminateContractsServiceOctopus.swift:14]: We need this when we get the implementation from the BE

**Review by julia-andersson — APPROVED** [review]

**Pull request merged**